### PR TITLE
Improve code readability for humans and IDEs

### PR DESCRIPTION
**Pull request by ravage84**

### CODE REVIEW

File: src/TestSuite/TestCase.php [hunk 1]
```diff
@@ -197,21 +197,22 @@ public function loadFixtures()
      * Useful to test how plugins being loaded/not loaded interact with other
      * elements in CakePHP or applications.
      *
-     * @param array $plugins list of Plugins to load
+     * @param array $plugins List of Plugins to load.
      * @return \Cake\Http\BaseApplication
      */
     public function loadPlugins(array $plugins = [])
     {
+        /** @var BaseApplication $app */
```
Comment (dereuromark) [inline src/TestSuite/TestCase.php]: Technically this is `|MockObject`
But as long as the mock is not used inside of the method, this is fine.

**Review by dereuromark — APPROVED** [review]

**Comment by markstory** [comment]
Looks good if you can convince the typecheckers that you know what you're doing.

File: src/TestSuite/TestCase.php (hunk 1 above)
Comment (ADmad) [reply src/TestSuite/TestCase.php]: Should use FQCN.

Comment (ravage84) [reply src/TestSuite/TestCase.php]: @dereuromark agree, but for this I focused on readability and on the actual relevant use.
@ADmad done.

File: src/TestSuite/TestCase.php [hunk 2]
```diff
@@ -197,21 +197,22 @@ public function loadFixtures()
      * Useful to test how plugins being loaded/not loaded interact with other
      * elements in CakePHP or applications.
      *
-     * @param array $plugins list of Plugins to load
+     * @param array $plugins List of Plugins to load.
      * @return \Cake\Http\BaseApplication
      */
     public function loadPlugins(array $plugins = [])
     {
+        /** @var \Cake\Http\BaseApplication $app */
```
Comment (saeideng) [inline src/TestSuite/TestCase.php]: after adding this line you can remove 
https://github.com/cakephp/cakephp/blob/bd2cee377d9b63f01a0a6cd443e580adfb588a10/phpstan.neon#L27 line

Comment (ravage84) [reply src/TestSuite/TestCase.php]: @saeideng done, thanks for pointing out.

**Pull request merged**